printPrimeNumbers.js

```diff
@@ -0,0 +1,18 @@
+let isPrime = true;
```

Comment:
Turn this into a function that receives one and return true if its a prime number, otherwise return false

Reply:
fixed on 373c405